DownFall/Scenes/Store/StoreScene.swift

```diff
@@ -301,19 +308,31 @@ extension StoreScene: StoreItemDelegate {
     }
 }
 
-extension StoreScene : ButtonDelegate {
+extension StoreScene: ButtonDelegate {
     func buttonPressed(_ button: Button) {
         if button.name == "leaveStore" {
             storeSceneDelegate?.leave(self, updatedPlayerData: playerData)
-        } else if button.name == buyButton,
+        } else if button.name == Constants.buyButton,
             let storeItem = selectedItem {
             buy(storeItem)
-        } else if button.name == sellButton,
+        } else if button.name == Constants.sellButton,
             let storeItem = selectedItem {
             sell(storeItem)
-        } else if button.name == closeButton {
+        } else if button.name == Constants.closeButton {
             selectedItem = nil
         }
 
     }
 }
+
+extension StoreScene: LabelDelegate {
+    func labelPressed(_ label: Label) {
+        
+    }
+    
+    func labelPressBegan(_ label: Label) {
+        
+    }
+    
+    
+}
```

Comment:
can we just pass in nil as a LabelDelegate?  Then we dont need to conform to this